common/views/themes/base/fonts.ts

```diff
@@ -5,7 +5,7 @@ export const fonts = css<GlobalStyleProps>`
   @font-face {
     font-family: 'Inter';
     font-style: normal;
-    font-weight: 400;
+    font-weight: 500;
```

Comment:
Had a chat with Jason and "normal" would usually be 500, lmk if it was wrong to change.